<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenPwDlg</class>
 <widget class="QDialog" name="GenPwDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Generator</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>442</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="tabCategory">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRandom">
      <attribute name="title">
       <string>Random</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Use following character groups:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox2">
             <property name="text">
              <string>&amp;Lower Letters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox7">
             <property name="text">
              <string>&amp;Underline</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox3">
             <property name="text">
              <string>&amp;Numbers</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox5">
             <property name="text">
              <string>&amp;White Spaces</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox1">
             <property name="text">
              <string>&amp;Upper Letters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox6">
             <property name="text">
              <string>&amp;Minus</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkBox4">
             <property name="text">
              <string>&amp;Special Characters</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="Check_ExcludeLookAlike">
         <property name="text">
          <string>Exclude look-alike characters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Check_EveryGroup">
         <property name="text">
          <string>Ensure that password contains characters from every group</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPronounceable">
      <attribute name="title">
       <string>Pronounceable</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="verticalSpacing">
          <number>12</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxPL">
           <property name="text">
            <string>Lower Letters</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxPU">
           <property name="text">
            <string>Upper Letters</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxPN">
           <property name="text">
            <string>Numbers</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxPS">
           <property name="text">
            <string>Special Characters</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustom">
      <attribute name="title">
       <string>Custom</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Use the following characters:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Edit_chars">
         <property name="font">
          <font>
           <family>Serif</family>
          </font>
         </property>
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Spin_Num">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel5">
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="Progress_Quali">
          <property name="maximum">
           <number>128</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="Check_CollectEntropy">
          <property name="text">
           <string>Enable entropy collection</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="Check_CollectOncePerSession">
          <property name="text">
           <string>Collect only once per session</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel4">
       <property name="text">
        <string>New Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit_dest">
       <property name="font">
        <font>
         <family>Serif</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ButtonChangeEchoMode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonGenerate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabCategory</tabstop>
  <tabstop>checkBox1</tabstop>
  <tabstop>checkBox2</tabstop>
  <tabstop>checkBox3</tabstop>
  <tabstop>checkBox5</tabstop>
  <tabstop>checkBox6</tabstop>
  <tabstop>checkBox7</tabstop>
  <tabstop>checkBox4</tabstop>
  <tabstop>Check_ExcludeLookAlike</tabstop>
  <tabstop>Check_EveryGroup</tabstop>
  <tabstop>Spin_Num</tabstop>
  <tabstop>Check_CollectEntropy</tabstop>
  <tabstop>Check_CollectOncePerSession</tabstop>
  <tabstop>Edit_dest</tabstop>
  <tabstop>ButtonChangeEchoMode</tabstop>
  <tabstop>ButtonGenerate</tabstop>
  <tabstop>checkBoxPL</tabstop>
  <tabstop>checkBoxPU</tabstop>
  <tabstop>checkBoxPN</tabstop>
  <tabstop>checkBoxPS</tabstop>
  <tabstop>DialogButtons</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
